pkgs/by-name/sh/shopify-cli_3/package.nix

```
@@ -0,0 +1,57 @@
+{ buildNpmPackage, lib, makeWrapper, bundlerEnv, testers, shopify-cli_3, ripgrep }:
+let
+  version = "3.63.2";
+
+  # Package the legacy ruby CLI.
+  rubyGems = bundlerEnv {
+    name = "shopify-cli-legacy";
+    gemdir = ./.;
+  };
+in
+buildNpmPackage {
+  pname = "shopify";
+  version = version;
+
+  src = lib.fileset.toSource {
+    root = ./.;
+    fileset = with lib.fileset; unions [
+      ./package.json
+      ./package-lock.json
+    ];
+  };
+
+  npmDepsHash = "sha256-6CEDcWXZXYHFrT2xpbj5NwMrbDZXH6HclgTGkfKDlJs=";
+  dontNpmBuild = true;
+
+  buildInputs = [ makeWrapper ];
+
+  passthru = {
+    updateScript = ./update.sh;
+    tests.version = testers.testVersion {
+      package = shopify-cli_3;
+      command = "shopify version";
+    };
+  };
+
+  postInstall = ''
+    # Disable the installCLIDependencies function.
+    substituteInPlace $(${ripgrep}/bin/rg 'await installCLIDependencies' $out/lib/node_modules/shopify/node_modules/@shopify/cli/dist --files-with-matches) \
```

```suggestion
    substituteInPlace $(rg 'await installCLIDependencies' $out/lib/node_modules/shopify/node_modules/@shopify/cli/dist --files-with-matches) \
```
Can we just use grep to avoid triggering more rebuilds than necessary when changing ripgrep?

---

Yes, I'll swap out `ripgrep` for `grep`. thx!